content/projects/git-exercises/_index.md

```diff
@@ -23,7 +23,7 @@ This little exercise will take you through the basic git mechanisms you need to
 
 **IMPORTANT** Make sure you go through the prerequisites listed above. Otherwise you'll be clueless!
 
-## Skillz you'll learn
+## Skills you'll learn
```

Comment:
@sagarmulchandani please revert this change as the spelling was intentional, as soon as this is done i can approve :)

Reply:
Done. You can go ahead with the review